src/account_management.rs

```
@@ -0,0 +1,163 @@
+use crate::contacts::initial_sync;
+use crate::{
+    ACCOUNTS_DIR, Config, ensure_accounts_dir, get_account_store_path, list_accounts, open_store,
+};
+use anyhow::{Result, anyhow};
+use presage::Manager;
+use std::path::Path;
+
+pub async fn create_account_cli(account_name: String, device_name: String) -> Result<()> {
+    use futures::{channel::oneshot, future};
+    use presage::libsignal_service::configuration::SignalServers;
+    use qr2term;
+
+    println!("Creating account '{}'...", account_name);
+
+    let accounts = list_accounts()?;
+    if accounts.contains(&account_name) {
+        return Err(anyhow::anyhow!("Account '{}' already exists", account_name));
+    }
+    ensure_accounts_dir()?;
+
+    let account_dir = format!("{}/{}", ACCOUNTS_DIR, account_name);
+    let store_path = get_account_store_path(&account_name);
+
+    tokio::fs::create_dir_all(&account_dir).await?;
+
+    let _ = std::fs::remove_file(&store_path);
```

I get it that it is not crucial and it can fail, but at least log this error with an `error()!` instead of completely ignoring it.